README.rst

```diff
@@ -36,14 +36,24 @@ The following versions of Python are supported:
 
 Basic Usage
 -------------
+You can expect a ``Client`` instance to have reasonable defaults, like the Fauna endpoint ``db.fauna.com`` and a global HTTP client, but you will always need to supply a secret.
+
+You can configure your secret by passing it directly to the client or by setting an environment variable.
+
+Supported Environment Variables:
+
+* ``FAUNA_ENDPOINT``: The Fauna endpoint to use. For example, ``http://localhost:8443``
+* ``FAUNA_SECRET``: The Fauna secret to use.
 
 .. code-block:: python
 
     from fauna import fql
     from fauna.client import Client
     from fauna.errors import FaunaException
 
-    client = Client()
+    # Assumes you have retrieved a secret from a secret store and set it to `my_secret`.
+    # As a best practice, don't store your secret directly in your code.
+    client = Client(secret=my_secret)
```

Comment:
I think we only demonstrate the use of `client = Client()` to encourage the use of the env var.

Reply:
Maybe this?

```suggestion
    # As a best practice, don't store your secret directly in your code.
    # Either set the FAUNA_SECRET, or retrieve it from a secret store.
    client = Client(secret=my_secret)
```